test/public/defaults.js

```diff
@@ -439,3 +439,14 @@ module.exports.checkMismatchingUrlParam = async (page, expectedUrlParameters) =>
     }
     return ret;
 };
+
+/**
+ * Wait For table to reload data
+ * Reloaded table MUST contain at least one row of data
+ * @param {puppeteer.Page} page the puppeteer page
+ * @param {function} triggerFunction table reload trigger
```

Comment:
Function called to trigger table data loading

Reply:
applied